utility/src/main/java/org/oppia/android/util/parser/html/MathTagHandler.kt

```diff
@@ -30,36 +33,48 @@ class MathTagHandler(
       attributes.getJsonObjectValue(CUSTOM_MATH_SVG_PATH_ATTRIBUTE)
     )
     if (content != null) {
-      // Insert an image span where the custom tag currently is to load the SVG. In the future, this
-      // could also load a LaTeX span, instead. Note that this approach is based on Android's Html
-      // parser.
-      val drawable =
-        imageRetriever.loadDrawable(
-          content.svgFilename,
-          CustomHtmlContentHandler.ImageRetriever.Type.INLINE_TEXT_IMAGE
-        )
       val (startIndex, endIndex) = output.run {
         // Use a control character to ensure that there's at least 1 character on which to "attach"
         // the image when rendering the HTML.
         val startIndex = length
         append('\uFFFC')
         return@run startIndex to length
       }
-      output.setSpan(
-        ImageSpan(drawable, content.svgFilename),
-        startIndex,
-        endIndex,
-        Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
-      )
+      if (content.svgFilename != null) {
+        // Insert an image span where the custom tag currently is to load the SVG. In the future, this
+        // could also load a LaTeX span, instead. Note that this approach is based on Android's Html
+        // parser.
+        val drawable =
+          imageRetriever.loadDrawable(
+            content.svgFilename,
+            CustomHtmlContentHandler.ImageRetriever.Type.INLINE_TEXT_IMAGE
+          )
+        output.setSpan(
+          ImageSpan(drawable, content.svgFilename),
+          startIndex,
+          endIndex,
+          Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+      } else if (content.rawLatex != null) {
+        output.setSpan(
+          MathExpressionSpan(content.rawLatex, 72f, assetManager, true),
```

Comment:
We need to come up with the base text height or might need a way to calculate and pass it here as an argument. 
Do we have any current implementation with the same use case, where we calculate the text height based on the screen for the content?

Reply:
use 16 sp